pyctcdecode/decoder.py

```diff
@@ -700,8 +700,7 @@ def parse_directory_contents(filepath: str) -> Dict[str, Union[str, None]]:
         contents.remove(BeamSearchDecoderCTC._ALPHABET_SERIALIZED_FILENAME)
         lm_directory: Optional[str]
         if contents:
-            lm_directory = contents[0]
```

Comment:
@mikeyshulman @gkucsko - would it maybe be possible to just make sure that `"language_model"` is present in the directory (along with the alphabet) and not force the directory to only contain the lm folder and the alphabet? 

This would make it much nicer and easier to save both a speech model and a decoder in one directory. E.g. this change would allow for this dir structure: 
https://huggingface.co/patrickvonplaten/wav2vec2-large-xlsr-53-spanish-with-lm/tree/main
. 

Without this relaxation, we would need to further nest the `language_model` directory in some kind of `decoder` directory which would then create 2 levels of nesting which IMO is not really necessary. 

Would that be ok for you?